src/plugins/home/server/capabilities_provider.ts

```diff
@@ -24,6 +24,6 @@ export const capabilitiesProvider = () => ({
     visualize: true,
     console: true,
     advanced_settings: true,
-    index_patterns: true,
+    indexPatterns: true,
```

Comment:
note: The NP migration for index pattern management changed the capability from `index_patterns` to `indexPatterns`, so this PR updates all references as appropriate.